assembl/static2/config.json

```diff
@@ -668,6 +668,12 @@
       "code": 59413,
       "src": "fontawesome"
     },
+    {
+      "uid": "dd6c6b221a1088ff8a9b9cd32d0b3dd5",
+      "css": "check",
+      "code": 59414,
+      "src": "fontawesome"
+    },
```

Comment:
You will need to regenerate the font. The font was edited in develop since then.